index.js

```diff
@@ -26,7 +26,7 @@ var exec = require('child_process').exec;
 var program = require('commander');
 var notifier = require('node-notifier');
 
-var pwd = require('process').env.PWD;
+var pwd = require('process').env.PWD || require('process').cwd();
```

Comment:
depends on the use-case. PWD is just the env variable which can be overwritten but cwd() returns the real current working directory. and yes, it works on all platforms.